hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSourceInterface.java

```diff
@@ -191,4 +194,17 @@ void addHFileRefs(TableName tableName, byte[] family, List<Pair<Path, Path>> pai
   default boolean isRecovered() {
     return false;
   }
+
+  /**
+   * ReplicationSources keep their replication queues and queue state out in the replication
+   * store which is external to the current process -- usually zookeeper -- in case of crash so
+   * another process can pick up where the crashed process left off. A few specialized replication
+   * sources intentionally avoid keeping state in the replication store because they reset and
+   * start over after a crash so need of recovery. One such is the replication of the hbase:meta
+   * table. Its ReplicationSource will return false when this method is called.
+   * @return True if this source persists its queues and queue state to the replication store.
+   */
+  default boolean isQueueStored() {
```

Comment:
Maybe 'persistent' is better then 'stored'?

Reply:
That is better.

Let me change. Also need to do another pass on peerid vs queueid... They get intermixed....Let me try and clean up some.

Reply:
Ugh, going to leave the cleanup for now. Hard to untangle. Just change the method name per your suggestion.